misk-aws/src/main/kotlin/misk/jobqueue/sqs/ForSqsConsumer.kt

```diff
@@ -3,4 +3,5 @@ package misk.jobqueue.sqs
 import javax.inject.Qualifier
 
 @Qualifier
-internal annotation class ForSqsConsumer
\ No newline at end of file
+@Target(AnnotationTarget.FIELD, AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
```

Comment:
hm, what necessitated this change (it's fine by me, just curious)

Reply:
so you could field inject in the test

```@Inject @ForSqsConsumer RepeatedTaskQueue```

Reply:
without having to do `@field:ForSqsConsumer`